CHANGELOG.md

```diff
@@ -105,6 +105,7 @@ We refer to [GitHub issues](https://github.com/JabRef/jabref/issues) by using `#
 - Fixed [#2104](https://github.com/JabRef/jabref/issues/#2104): Crash after saving BibTeX source with parsing error
 - Fixed [#2109](https://github.com/JabRef/jabref/issues/#2109): <kbd>Ctrl-s</kbd> doesn't trigger parsing error message
 - Fixed RTFChars would only use "?" for characters with unicode over the value of 127, now it uses the base character (é -> e instead of ?)
+- Fixed [#2177](https://github.com/JabRef/jabref/issues/#2177): NullPointerException caused by clearing bibtexkey field in entry editor
```

Comment:
Is this bug already present in version 3.6? If not, then please remove the changelog entry since in the changelog we only want to state what changed relative to the previously released version.